test/reputation-system/happy-paths.js

```diff
@@ -264,11 +264,11 @@ contract("Reputation Mining - happy paths", (accounts) => {
 
       // Complete two reputation cycles to process the log
       await advanceMiningCycleNoContest({ colonyNetwork, test: this });
-      goodClient = new ReputationMinerTestWrapper({ loader, realProviderPort, useJsTree: false, minerAddress: MINER1 });
-      await goodClient.initialise(colonyNetwork.address);
-      await goodClient.resetDB();
+      const goodClientSolidityTree = new ReputationMinerTestWrapper({ loader, realProviderPort, useJsTree: false, minerAddress: MINER1 });
```

Comment:
Ah, so we're using a different constant here to avoid accidentally sharing the state between tests?

Reply:
Yeah, the global `goodClient` is now not reassigned anywhere.